mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
@laurynas-biveinis This is how other Seconds_Behind_Master tests are constructed. See rpl_seconds_behind_master.test. Upper bound is not arbitrary high - it is time between just before applying changes on master and the moment we are checking SBM on slave - so it is bounded. As to point 2) real_sleep 1 - is also deterministic. MTS checkpoint which refreshes SBM is to be executed every 300ms. Thus if after 1 sec SBM is not updated it means that somehting is broken and the test should fail.